docs/install_guides/installing-openvino-overview.md

```diff
@@ -42,9 +42,9 @@ For Python developers, the easiest way is to [install OpenVINO Development Tools
 
 For C++ developers, you may choose one of the following installation options for OpenVINO Runtime on your specific operating system:
 
-* Linux: You can install OpenVINO Runtime using an [Installer](installing-openvino-linux.md), [APT](installing-openvino-apt.md), [YUM](installing-openvino-yum.md), [Anaconda Cloud](installing-openvino-conda.md), or [Docker](installing-openvino-docker-linux.md).
-* Windows: You can install OpenVINO Runtime using an [Installer](installing-openvino-windows.md), [Anaconda Cloud](installing-openvino-conda.md), or [Docker](installing-openvino-docker-windows.md).
-* macOS: You can install OpenVINO Runtime using an [Installer](installing-openvino-macos.md) or [Anaconda Cloud](installing-openvino-conda.md).
+* Linux: You can install OpenVINO Runtime using a [Docker](installing-openvino-docker-linux.md).
```

Comment:
```suggestion
* Linux: You can install OpenVINO Runtime using archive files or Docker. See [Install OpenVINO on Linux](installing-openvino-linux-header.md).
```
Let's apply the same approach for the other two OSes for now.

Reply:
Windows & macOS done in https://github.com/openvinotoolkit/openvino/pull/12222/commits/68b30eedc62a3d71865d19f5d119de9820b794ef